test/multicall/Multicall.js

```diff
@@ -0,0 +1,328 @@
+/*-
+ *
+ * Hedera JSON RPC Relay - Hardhat Example
+ *
+ * Copyright (C) 2022 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+const {expect} = require("chai");
+const {ethers} = require('hardhat');
+const {solidityPack} = require("ethers/lib/utils");
+const {defaultAbiCoder} = require("@ethersproject/abi");
+
+describe('Multicall3', function () {
+
+    let multicaller, receiver, reverter;
+
+    // Translates to: "Multicall3: call failed"
+    const REVERT_REASON_DATA = '0x08c379a0000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000174d756c746963616c6c333a2063616c6c206661696c6564000000000000000000';
+    const RESULT_FIVE = '0x0000000000000000000000000000000000000000000000000000000000000005';
+    const INPUT_ELEMENT_LENGTH = 266;
```

Comment:
ok!  (now my lack of encoding knowledge shows up: I can account for 264 bytes = selector + 4 256-bit params, but where does the last (encoded) byte come from?)

(you could if you wanted add a comment here: selector + 4 params + ? (that last byte))